.jscsrc

```diff
@@ -0,0 +1,45 @@
+{
+    "requireOperatorBeforeLineBreak": true,
+    "maximumLineLength": {
+      "value": 100,
+      "allowComments": true,
+      "allowRegex": true
+    },
+    // disabled due to https://github.com/jscs-dev/node-jscs/issues/915
+    // "validateIndentation": 2,
```

Comment:
Think this will be a tough one to fix? I'd like to also remove `indent` from our `.jshintrc`, but we could do it in a future commit if it's non-trivial.